src/tests/ftest/util/test_utils_pool.py

```diff
@@ -1422,3 +1424,49 @@ def measure_rebuild_time(self, operation, interval=1):
         self.wait_for_rebuild_to_end(interval=interval)
         duration = time() - start
         self.log.info("%s duration: %.1f sec", operation, duration)
+
+    def check_pool_files(self, log, hosts, uuid, scm_mount):
+        """Check if pool files exist on the specified list of hosts.
+
+        Args:
+            log (logging): logging object used to display messages
+            hosts (NodeSet): list of hosts
+            uuid (str): uuid file name to look for in /mnt/daos.
+            scm_mount (str): SCM mount point such as "/mnt/daos". From test, it can be
+                obtained as self.server_managers[0].get_config_value("scm_mount")
+
+        Returns:
+            bool: True if the files for this pool exist on each host; False
+                otherwise
+
+        """
+        status = True
+        log.info("Checking for pool data on %s", hosts)
+        pool_files = [uuid, "superblock"]
+        for filename in [f"{scm_mount}/{item}" for item in pool_files]:
+            result = check_file_exists(hosts, filename, sudo=True)
+            if not result[0]:
+                log.error("%s: %s not found", result[1], filename)
+                status = False
+        return status
+
+    def check_for_pool(self, host, uuid, scm_mount):
+        """Check if pool folder exist on server.
+
+        Args:
+            host (NodeSet): Server host name
+            uuid (str): Pool uuid to check if exists
+            scm_mount (str): SCM mount point such as "/mnt/daos". From test, it can be
+                obtained as self.server_managers[0].get_config_value("scm_mount")
+
+        Returns:
+            bool: True if pool folder exists, False otherwise
+
+        """
+        pool_dir = f"{scm_mount}/{uuid}"
+        result = check_file_exists(host, pool_dir, directory=True, sudo=True)
+        if result[0]:
+            print("{} exists on {}".format(pool_dir, host))
+        else:
+            print("{} does not exist on {}".format(pool_dir, host))
```

Comment:
We should replace `print` with `self.log`

Reply:
Done